eodag/rest/utils.py

```diff
@@ -1016,8 +1015,14 @@ def rename_to_stac_standard(key: str) -> str:
     stac_config_properties = stac_config["item"]["properties"]
 
     for stac_property, value in stac_config_properties.items():
+        if isinstance(value, list):
+            value = value[0]
         if str(value).endswith(key):
             return stac_property
+
+    if key in OSEO_METADATA_MAPPING:
```

Comment:
use `get_criterias_from_metadata_mapping(stac_provider_metadata_mapping, arguments)` for that